src/proxy.c

```diff
@@ -47,6 +47,7 @@ struct watcher_proxy_data
 	 * otherwise it WILL wait
 	 */
 	int         *oob_fd;
+	gboolean    oob_fd_found;
```

Comment:
Yes, but I need to carry it through 2 `cc_proxy_msg_read()` calls. Once we find the OOB data, we return from this routine and get called again when the actual data comes in. On the second run we want to skip the OOB check.